brother_ql/brother_ql_print.py

```diff
@@ -55,8 +55,10 @@ def main():
         try:
             selected_backend = guess_backend(args.printer)
         except:
-            logger.info("No backend stated. Selecting the default linux_kernel backend.")
-            selected_backend = 'linux_kernel'
+            logger.info(
+                "No backend stated. Selecting the default pyusb backend."
+            )
```

Comment:
Q: Why is the default backend changed?

Reply:
I've explained it in the commit message, pyusb is cross-platform and works out of the box (after the correct udev
rules have been set up on Linux), while `/dev/usb/lp0` is still typically inaccessible by regular users unless the user is in a specific user group. It also makes brother_ql work by default on Windows and macOS, since `linux_kernel` is obviously limited to Linux.

Reply:
I will look at the implementations currently using brother_ql-inventree and brother_ql and see how widespread this is explicitly set to linux_kernel via guess_backend and then decide if this is reasonable or if we need a deprecation thingy.

Reply:
My reasoning for this change is that on Windows or macOS, people would already be using pyusb anyway, so this will not affect them at all. I'm not sure how many people are using the `linux_kernel` backend on Linux, because setting up a udev rule seems easier to me compared to making sure every user who needs printer access is a part of the `lp` group.

Reply:
There seem to be quite a lot of scripts using `file:` as an address. The default should not be changed. As far as my testing shows, this PR works either way so my last request would be to change this back to defaulting to linux_kernel.

Reply:
I've reverted all the changes I've made, however the existing codebase already has some inconsistent behavior where `pyusb` is still used as the default. Those have been left alone.